src/components/Questionnaire/QuestionTypes/MedicationQuestion.tsx

```
@@ -0,0 +1,576 @@
+          variant="outline"
+          size="sm"
+          className="mt-4"
+          onClick={handleAddMedication}
+          disabled={disabled}
+        >
+          <PlusIcon className="mr-2 h-4 w-4" />
+          Add Medication
+        </Button>
+      </div>
+    </div>
+  );
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)